packages/twenty-front/src/modules/workflow/utils/getWorkflowVersionDiagram.ts

```diff
@@ -0,0 +1,28 @@
+import { WorkflowVersion } from '@/workflow/types/Workflow';
+import { WorkflowDiagram } from '@/workflow/types/WorkflowDiagram';
+import { generateWorkflowDiagram } from '@/workflow/utils/generateWorkflowDiagram';
+import { isDefined } from 'twenty-ui';
+
+const EMPTY_DIAGRAM: WorkflowDiagram = {
+  nodes: [],
+  edges: [],
+};
+
+export const getWorkflowVersionDiagram = (
+  workflowVersion: WorkflowVersion | undefined,
+): WorkflowDiagram => {
+  if (
+    !(
+      isDefined(workflowVersion) &&
+      isDefined(workflowVersion.trigger) &&
```

Comment:
We may want to allow the diagram to be displayed if the trigger is defined but not the steps. For now, I can't assure you it's supported, so I think it's better to disallow this edge case.